voice_servers/teamspeak_ARM64/egg-teamspeak3-arm64-server.json

```diff
@@ -13,7 +13,7 @@
         "ghcr.io\/parkervcp\/yolks:box64": "ghcr.io\/parkervcp\/yolks:box64"
     },
     "file_denylist": [],
-    "startup": "box64 .\/ts3server default_voice_port={{SERVER_PORT}} query_port={{QUERY_PORT}} filetransfer_ip=0.0.0.0 filetransfer_port={{FILE_TRANSFER}} license_accepted=1",
+    "startup": "box64 .\/ts3server default_voice_port={{SERVER_PORT}} query_port={{QUERY_PORT}} filetransfer_ip=0.0.0.0 filetransfer_port={{FILE_TRANSFER}} serveradmin_password={{SERVERADMIN_PASSWORD}} license_accepted=1",
```

Comment:
Does it work if the SERVERADMIN_PASSWORD variable is empty?

Reply:
in the env stands that is required but if i set it to nullable As you suggested, then the server starts and when you try to connect to yatqa without password you still have no rights
sou i guess its fine